### ISSUE TITLE
Image slider v3.2.0 - position of the slides are not saved

### ISSUE THREAD

**Issue by PeNov** [PrestaShop/PrestaShop#37027]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

Hello,
after moving slides in BO (module Image slider), the new position of the slides are not saved.

Thank you
BR
Petr

### Steps to reproduce

1. Go to module Image slider.
2. Move slides and check new position in DB (ps_homeslider_slides). 

### PrestaShop version(s) where the bug happened

8.1.6

### PHP version(s) where the bug happened

7.4.33

### If your bug is related to a module, specify its name and its version

Image slider, v3.2.0

**Comment by Hlavtox** [comment]
@PeNov Hi! :-) Can you please open developer tools (F12), move the slider and tell me what it says on Console and network tab?

If there is a response on the AJAX, can you check what is on the "Response" tab if you click on it?

**Comment by PeNov** [comment]
Hi @Hlavtox 
Console:
![console](https://github.com/user-attachments/assets/3831d035-03af-4579-9478-d0a30f7d61f2)
Network:
![network](https://github.com/user-attachments/assets/4b5a68d0-1df1-4ab3-af15-63243ae09352)

Response after move slide:
![move_slide](https://github.com/user-attachments/assets/32d943b4-7a0e-44e7-b505-dc7e5f8ccc95)

**Comment by Hlavtox** [comment]
@PeNov I managed to reproduce the issue.

The module is getting completely nonsense data to update the slides. :-)))

```
Array
(
    [0] => 0
    [1] => 2
    [2] => 3
    [3] => 8
    [4] => 2
    [5] => 6
    [6] => 4
    [7] => 1
    [8] => 9
    [9] => 7
    [10] => 0
    [11] => 7
    [12] => 2
    [13] => 1
    [14] => 9
    [15] => 9
    [16] => 6
    [17] => 3
    [18] => 5
    [19] => 5
    [20] => 8
    [21] => 7
    [22] => 7
    [23] => 4
    [24] => 8
    [25] => 1
    [26] => 6
)

```

**Comment by PeNov** [comment]
Hi @Hlavtox 
I tested this module on the same version of PS, on the same server, only a different e-shop and works for 4 slides. For now, I'll fix it manually in the database.
Thank you

**Comment by Hlavtox** [comment]
@PeNov Please apply this fix, it will be in 3.2.1 :-) https://github.com/PrestaShop/ps_imageslider/pull/110

**Comment by PeNov** [comment]
@Hlavtox Great, fix works perfectly. Thank you
